recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 source:
   fn: onig-{{ version }}.tar.gz
   url: https://github.com/kkos/oniguruma/releases/download/v{{ version }}/onig-{{ version }}.tar.gz
-  sha256: adeada5f6b54c2a6f58ff021831a01b18a62b55ea9935e972e36ebb19e7c4903
+  sha256: 91bfb25e050ce3c301483204dd3f0d03a7c05472e20d48fe227a383d4534e7c9
 
 build:
-  number: 1001
+  number: 0
```

Comment:
Does this need to be bumped back to 1000?

cc @conda-forge/core

Reply:
Yes most likely. I guess we could put an escape hatch in the bot to detect if it is 1000+ and then drop it back to 1000.

Reply:
Raised issue ( https://github.com/regro/cf-scripts/issues/360 ) on this point.